webpack.config.js

```diff
@@ -1,40 +1,40 @@
 const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 
-module.exports = { 
-    entry: {
-        index: './src/index.js',
-        // print: './src/print.js',
-      },   
-      devtool: 'inline-source-map',
-      devServer: {
-        static: './dist',
-      },
-    
-      plugins: [
-        new HtmlWebpackPlugin({
-        //   title: 'Output Management',
-          template: './src/index.html',
-        }),
-      ],
-      output: {
-        filename: '[name].bundle.js',
-        path: path.resolve(__dirname, 'dist'),
-      },
-      optimization: {
-        runtimeChunk: 'single',
-      },
-      module: {
-        rules: [
-          {
-            test: /\.css$/i,
-            use: ['style-loader', 'css-loader'],
-          },
-        ],
+module.exports = {
+  entry: {
+    index: './src/index.js',
+    // print: './src/print.js',
+  },
+  devtool: 'inline-source-map',
+  devServer: {
+    static: './dist',
+  },
+
+  plugins: [
+    new HtmlWebpackPlugin({
+      //   title: 'Output Management',
+      template: './src/index.html',
+    }),
+  ],
+  output: {
+    filename: '[name].bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+  optimization: {
+    runtimeChunk: 'single',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.css$/i,
+        use: ['style-loader', 'css-loader'],
       },
-//   output: {
-//     filename: 'main.js',
-//     path: path.resolve(__dirname, 'dist'),
-//   },
+    ],
+  },
+  //   output: {
+  //     filename: 'main.js',
+  //     path: path.resolve(__dirname, 'dist'),
+  //   },
```

Comment:
- [ ] Please remove this commented code because I believe it is bad practice to leave comment code like this in your codebase and it makes your code look unclean.🤓 🛠️
- [ ] And as well please remove what you have commented at `line 16` and `line 7`. 👌